lib/context_adapter_server.js

```diff
@@ -363,15 +371,27 @@ function stop(callback) {
     operationType: caConfig.OPERATION_TYPE.SERVER_STOP
   };
 
-  caLogger.info(context, 'Stopping the Context Adapter server...');
+  caLogger.info(
+    context,
+    'Stopping the Context Adapter server...'
+  );
+
   if (server && server.info && server.info.started) {
     server.stop(function(err) {
       // Server successfully stopped
-      caLogger.info(context, 'hapi server successfully stopped');
+      caLogger.info(
+        context,
+        'hapi server successfully stopped'
```

Comment:
consider something like "Http server (hapi) successfully stopped" since you are the only person using hapi